dask_jobqueue/oar.py

```diff
@@ -76,6 +80,42 @@ def __init__(
         # Add extra header directives
         header_lines.extend(["#OAR %s" % arg for arg in self.job_extra_directives])
 
+        # Memory
+        memory = self.worker_memory
+        if memory is not None:
+            if oar_mem_core_property_name is None:
+                oar_mem_core_property_name = dask.config.get(
+                    "jobqueue.%s.oar-mem-core-property-name" % self.config_name
+                )
+            if oar_mem_core_property_name is None:
+                warn = (
+                    "The OAR property name corresponding to the memory per core of your cluster has not been set. "
+                    "Thus, the memory parameter will not be used by OAR. "
+                    "It can be set through oar_mem_core_property_name, e.g., memcore, mem_core.. "
+                )
+                warnings.warn(warn, category=UserWarning)
```

Comment:
Je pense que le warning n'est pas bon, il faut être plus factuel : 

'oar_mem_core_property_name is not set thus OAR will not take the memory parameter into account'